common/build-style/cargo.sh

```diff
@@ -0,0 +1,33 @@
+#
+# This helper is for building rust projects which use cargo for building
+#
+
+do_configure() {
+	mkdir -p .cargo
+	# Don't be verbose, respect makejobs
+	cat >> .cargo/config <<EOF
+[term]
+verbose = false
+[build]
+jobs = ${makejobs#*j}
```

Comment:
Jobs can be applied as `-j XXX` to `cargo build`

Reply:
Yeah, but doing it here is easy enough and we don't have to supply them twice this way